src/endpoints/tokens/token.service.ts

```diff
@@ -838,6 +838,21 @@ export class TokenService {
       token => token.transactions ?? 0,
     );
 
+    const egldIdentifier = 'EGLD-000000';
+    const egldToken = new TokenDetailed({
+      identifier: egldIdentifier,
+      name: 'EGLD',
+      type: TokenType.FungibleESDT,
+      assets: await this.assetsService.getTokenAssets(egldIdentifier),
```

Comment:
same here

Reply:
here we will keep the identifier EGLD-000000